StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -43,6 +45,8 @@ class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewCo
 
         }
     }
+    // Edge-case, only populated when Link is selected via wallet in flow controller
+    var walletPaymentOption: PaymentSheet.PaymentOption?
```

Comment:
Let's name this more specifically like `var isLinkWalletButtonSelected: Bool`?  

Also think the type shouldn't be `PaymentOption`, since that means this could be like `.new(..)` or `.saved` which doesn't make sense.